src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,103 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.commands.Command;
+import javafx.application.Platform;
+import javafx.scene.image.Image;
+
+
+/**
+ * Container for Duke program.
+ */
+public class Duke {
+    private static final String FILE_PATH = "data/tasks";
+
+    private TaskList tasks;
+    private Ui ui;
+    private Storage storage;
+
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.JPG"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.JPG"));
+
+    /**
+     * Instantiates a Duke object.
+     *
+     * @param filePath String of file path of the previously saved task.txt file (if any).
+     */
+    public Duke(String filePath) {
```

Comment:
Shouldn't there be a `throws IOException` here?

Reply:
I used try-catch in the method body so didn't declare `throws IOException` in method header